edgecast/resources/cps/certificate.go

```diff
@@ -422,8 +422,53 @@ func ResourceCertificateRead(ctx context.Context,
 	d *schema.ResourceData,
 	m interface{},
 ) diag.Diagnostics {
-	// Not Yet Implemented
 	var diags diag.Diagnostics
+
+	config := m.(**api.ClientConfig)
+	cpsService, err := buildCPSService(**config)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	params := certificate.NewCertificateGetParams()
+	n, err := strconv.ParseInt(d.Get("id").(string), 10, 64)
+	if err == nil {
+		fmt.Printf("%d of type %T", n, n)
```

Comment:
Should this be an err != nil check and then print the ID below the check (if we want that printed out)?